docs/content/apis/dmf_api.md

```diff
@@ -63,11 +63,19 @@ Payload Template (optional):
 
 ```json
 {
-    "name": "String"
+    "name": "String",
+    "attributeUpdate": {
+        "attributes": {
+            "exampleKey1" : "exampleValue1",
+            "exampleKey2" : "exampleValue2"
+        },
+        "mode": "String"
+    }
 }
 ```
 
-The "name" property specifies the name of the thing, which by default is the thing ID. This property is optional.
+The "name" property specifies the name of the thing, which by default is the thing ID. This property is optional.<br />
+The "attributeUpdate" property provides the attributes of the thing, for details see UPDATE_ATTRIBUTES message. This property is optional.
```

Comment:
Please also document the "mode" property here... you can copy from UPDATE_ATTRIBUTES below...